public/content/lessons/2018/fourier-transforms/index.mdx

```diff
@@ -6,4 +6,343 @@ video: spUNpyF58BY
+
+Why are there $6$ petals on this winded up graph?
+
+<FreeResponse>
+
+The product of $\frac13$ and $\frac12$ is $\frac16$. If we had a winding frequency of $0.75$ cycles per second, there would be $\frac13\cdot\frac34=\frac14$, so 4 petals!
+
+</FreeResponse>
+
+It is important to see there are 2 different frequencies at play.
+
+1. The frequency of the original signal, 3 beats per second.
+2. The frequency that we are wrapping the sine wave along the circle, which at the moment is at 0.5 cycles per second.
+
+If we set the vector rotating faster or slower, changing the "winding frequency", the resulting shape it traces out would be something different.
+
+<Figure
+  image="3freq2.png"
+  video="3freq2.mp4"
+  width="100%"
+/>
+
+The key intuition here is that we are wrapping the signal around a circle, and depending on how tightly we wind the sine wave, we can find different patterns. So what happens when the winding frequency is *the same* as the original signal's frequency? When our little vector rotates around the circle at 3 cycles per second? Well, we get the following plot:
+
+<Figure
+  image="winding_match.png"
+  video="winding_match.mp4"
+  width="100%"
+/>
+
+Notice, the resulting curve it draws out is off-balance to the right. The rotating vector is always longer when it's pointing to the right, and shorter when it's pointed to the left, because the frequency with which its length changes is identical to the frequency with which it rotates around the circle.
+
+Would we be able to use this information for our frequency unwinding machine? Well, we actually can!
+
+First, let's imagine that the pattern was made of something with some weight, like a metal wire. Now put a dot at the center of mass location, and as we change the winding frequency, the center of mass will kind of wobble about. We'll revisit what we mean by "center of mass" here a bit further down, but for the moment we're looking for an intuitive way to measure how much this wound up graph is off-balance.
+
+In most cases, the center of mass stays relatively close to the origin, but when the winding frequency is the same as the frequency for our signal, the center of mass is unusually far to the right. To keep track of this effect, let's draw the x-coordinate for the center of mass as the winding frequency changes.
+
+<Figure
+  image="com_1.png"
+  video="com_1.mp4"
+  width="100%"
+/>
+
+Different patterns appear as we wind up this graph, but it is clear that the $x$-coordinate for the center of mass is important when the winding frequency is $3$. Check out how the red dot marking the center of mass moves as the winding frequency changes. The interactive applet below will respond to hovering your mouse over the plot of the $x$-coordinate.
+
+<Interactive filename="basic_interactive/index" aspectRatio={880 / 500} />
```

Comment:
I think the speed of the scrubbing also is related to the slow animations I discovered while watching Grant's screen. I realized now that the draw function is just called instantly after the previous draw has completed. I thought it was called at some fixed framerate, but it executes much faster on my computer than it did on Grant's. So I'll change the code so that the scrubbing is time based instead of number of draw frames. That way the scrubbing will be the same time for every device. Then we can pick a momentum value that is more snappy.